### PR TITLE
Added logout page

### PR DESCRIPTION
**Pull request by athar05**
image

### CODE REVIEW

File: logout.html [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+          <input class="search-bar" type="text" placeholder="Search..">
+          <a href="./login.html">
+            <div class="button button-icon"> <span> Login </span>
+            </div>
+          </a>
+        </div>
+
+        <div class="nav-right">
+          <figure class="icon icon-cart ">
+            <div> <i class="fa-solid fa-cart-shopping"></i></div>
```
Comment (Chiragmodi01) [inline logout.html:41]: why have you put them in div?

File: styles.css [hunk 2]
```diff
@@ -536,3 +536,58 @@ h4 {
   cursor: pointer;
 }
 /* login paged ends */
+
+/* logout page begins */
+.page-container.logout {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-template-rows: 80px 70vh 200px;
```
Comment (Chiragmodi01) [inline styles.css:544]: you're using three different units.

File: logout.html [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+<body class="page-container logout">
+  <div class="header">
+    <div class="header">
+      <div class="nav">
+        <div class="nav-left">
+          <a href="/index.html"> <img class="hero-logo" src="./images/running-shoes.png" alt=""></a>
+          <input class="search-bar" type="text" placeholder="Search..">
+          <a href="./login.html">
+            <div class="button button-icon"> <span> Login </span>
```
Comment (sashaboi) [inline logout.html:34]: adding a display flex, align items center , justifycontent center might help align the login text to the button.

File: styles.css [hunk 4]
```diff
@@ -536,3 +536,58 @@ h4 {
...
+}
+
+.page-container.logout .logout-section {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: 17vh;
+}
+
+.logout-section .text-md.text-primary.text-underline:hover {
```
Comment (Chiragmodi01) [inline styles.css:589]: this could be utility class.

**Review by Chiragmodi01 — APPROVED** [review]
you could just show toast for logout or make the whole page and not just box.

overall looks good!

**Review by sashaboi — APPROVED** [review]
Suggested a change for the nav bar button, looks good!

File: logout.html (hunk 1 above)
Comment (athar05) [reply logout.html:41]: I had made this commit in dev, will rectify it

File: styles.css (hunk 2 above)
Comment (athar05) [reply styles.css:544]: Will address it, thanks

File: logout.html (hunk 3 above)
Comment (athar05) [reply logout.html:34]: Thanks for bringing it to my notice. I had this class already, just hadn't added it.

**Pull request merged**